proc/proc.go

```diff
@@ -0,0 +1,32 @@
+package proc
+
+import (
+	"github.com/scootdev/scoot/runner"
+)
+
+type JobType int
```

Comment:
Thrift object should be used only for serialization and RPC, so as soon as you get a Thrift object, you should convert to a domain object.

Where do you see jobtype as a string?